packages/ui/components/ui/tx/view/value.tsx

```diff
@@ -17,7 +18,7 @@ export const ValueViewComponent = ({ view }: ValueViewPrpos) => {
     const encodedAssetId = bech32AssetId(value.assetId!);
     return (
       <div className='flex font-mono'>
-        <p className='text-[15px] leading-[22px]'>{fromBaseUnitAmount(amount, 1).toFormat()}</p>
+        <p className='text-[15px] leading-[22px]'>{fromBaseUnitAmount(amount, 0).toFormat()}</p>
```

Comment:
I'm like... 90% sure that using `1` for the default exponent for an unknown denomination is a mistake. An exponent of `1` means we're multiplying the `amount` by "10 to the power of 1" — i.e., we're multiplying it by 10. 

We _should_ (right?!) be multiplying it by `1`  (AKA, "10 to the power of 0") when we don't know the denomination.

Reviewers, please check my logic!

Reply:
i believe this is correct, but i would confirm by viewing a value through this and then comparing it with the same value viewed with pcli

Reply:
Ah good catch!